common/src/main/java/com/wynntils/mc/EventFactory.java

```diff
@@ -240,6 +242,11 @@ public static ContainerSetContentEvent onContainerSetContent(ClientboundContaine
                 packet.getItems(), packet.getCarriedItem(), packet.getContainerId(), packet.getStateId()));
     }
 
+    public static void onContainerSetSlot(ClientboundContainerSetSlotPacket packet) {
+        post(new ContainerSetSlotEvent(
```

Comment:
Is this still used? It does not seem so? Was it not needed once you started using WynnItemStacks, or is there some bug hiding here? 

And if it is not needed, should you remove it for the PR or do you think it will be needed very soon so you want to keep it?

Reply:
I want to keep it for my upcoming PRs